app/views/lessons/show.html.erb

```diff
@@ -31,71 +12,45 @@
+      </div>
     </div>
+    
+  </div>
 
-    <div class="action-box">
-      <div class="lc-end-wrapper">
-        <%= render 'shared/complete_lesson' %>
-      </div>
+  <div id="lesson-details">
+    <div class="container">
+      <%= render 'courses/course/banner', course: @course %>
+      <h2 class="text-center accent"><%= @lesson.title %> </h2>
 
-      <div class="lnav">
+      <%= link_to course_path(@lesson.course.title_url, :ref => "lnav"), :title => "Go back to \"#{@lesson.course.title}\""  do %>
+        <button class="button primary">View Course</button>
+      <% end %>
 
-        <div class="lnav-prev lnav-wrap <%= 'lnav-disabled' unless @lesson.prev_lesson %> ">
-          <% if @lesson.prev_lesson %>
-            <%= link_to course_lesson_path(@lesson.course, @lesson.prev_lesson, :ref => "lnav"), :title => "Go back to \"#{@lesson.prev_lesson.title}\""  do %>
-              <div class="lnav-prev-icon icon"></div>
-              <div class="lnav-text">
-                <p>Prev</p><p>Lesson</p>
-              </div>
-            <% end %>
-          <% else %>
-            <div class="lnav-prev-icon icon"></div>
-            <div class="lnav-text">
-              <p>Previous</p><p>Lesson</p>
-            </div>
-          <% end %>
-        </div>
+      <%= render 'mark_lesson_completed' %>
 
-        <div class="lnav-index lnav-wrap">
-          <%= link_to course_path(@lesson.course.title_url, :ref => "lnav"), :title => "Go back to \"#{@lesson.course.title}\""  do %>
-            <div class="lnav-course-icon icon <%= 'lnav-active' if user_signed_in? && current_user.has_completed?(@lesson) && @lesson.next_lesson.nil? %>"></div>
-            <div class="lnav-text">
-              <p>View</p><p>Course</p>
-            </div>
-          <% end %>
-        </div>
+      <%= link_to course_lesson_path(@lesson.course, @lesson.next_lesson, :ref => "lnav"), :title => "Move on to \"#{@lesson.next_lesson.title}\"" do %>
+        <button class="button primary">Next Lesson</button>
+      <% end %>
 
-        <div class="lnav-next lnav-wrap <%= 'lnav-disabled' unless @lesson.next_lesson %>">
-          <% if @lesson.next_lesson %>
-            <%= link_to course_lesson_path(@lesson.course, @lesson.next_lesson, :ref => "lnav"), :title => "Move on to \"#{@lesson.next_lesson.title}\"" do %>
-              <div class="lnav-next-icon icon <%= 'lnav-active' if user_signed_in? && current_user.has_completed?(@lesson) && !@lesson.next_lesson.nil? %>"></div>
-              <div class="lnav-text">
-                <p>Next</p><p>Lesson</p>
-              </div>
-            <% end %>
-          <% else %>
-            <div class="lnav-next-icon icon"></div>
-            <div class="lnav-text">
-              <p>Next</p><p>Lesson</p>
-            </div>
-          <% end %>
-        </div>
+      <div class="card">
+        <h3>Solutions</h3>
       </div>
+
+      <h1 class="text-center">Got Questions?</h1>
```

Comment:
Please use the `bottom_cta` partial. Have a look at [this](https://github.com/TheOdinProject/theodinproject/blob/redesign/app/views/static_pages/faq.html.erb#L10) for an example